web/js/utils-0.0.10.js

```diff
@@ -807,10 +807,20 @@
                                 break;
                             case 50: // 2
                                 if (that.symbol) {
-                                    that.unhighlight(that.symbol).length === 0 && that.highlight(that.symbol);
+                                    that.unhighlight(that.symbol).length === 0 && that.highlight(that.symbol, 'yellow');
```

Comment:
naming the styles in color-agnostic way would be better I think (second/third/... comes to mind)